rpc/baremetalhostagent/admin/credentialmonitor/moc_baremetalhostagent_credentialmonitor.proto

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/baremetalhostagent/admin";
+package moc.baremetalhostagent.admin;
+
+enum CertificateStatus {
+	Single = 0;
+	Overlap = 1;
+}
+
+message CredentialMonitorRequest {
+	CredentialMonitor CredentialMonitor = 1;
+}
+
+message CredentialMonitorResponse {
+    CredentialMonitor CredentialMonitor = 1;
+    string  Error = 2;
+}
+
+message CredentialMonitor {
+    bytes certificate = 1 [deprecated = true];
+    CertificateStatus status = 2; 
+    string newCertificate = 3;
+}
+
+service CredentialMonitorAgent {
```

Comment:
Whats the feature/requirement here.

Reply:
No requirement/feature at the moment. I removed this.